src/eventhubs/azext_eventhub/_params.py

```diff
@@ -0,0 +1,159 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core.commands.parameters import (tags_type, get_enum_type, resource_group_name_type)
+
+
+# pylint: disable=line-too-long
+def load_arguments_namespace(self, _):
+    with self.argument_context('eventhubs namespace exists') as c:
+        c.argument('namespace_name', options_list=['--name'], help='name of the Namespace')
```

Comment:
Like in SB extension, add `-n` as short option.

Reply:
Resolved